docs/madr/decisions/037-configure-all-gateways.md

```diff
@@ -0,0 +1,64 @@
+# Configuring many gateways
+
+- Status: accepted
+
+## Context and Problem Statement
+
+This document aims to clarify how to configure all gateways or a subset of them, especially when adjustments, such as in `MeshTimeout`, are needed compared to regular Mesh traffic.
+
+## Decision Drivers
+
+- When we set up default rules, we want to use different settings for gateways than we do for regular Mesh applications.
+
+## Considered Options
+
+- Introduce new kind `MeshGatewaysSubset`
+- Add label to all gateways and target them by kind `MeshSubset`
+
+## Decision Outcome
+
+- ?
+
+## Introduce new kind `MeshGatewaysSubset`
+
+We can introduce a new kind `MeshGatewaysSubset` It only picks gateways based on tags, if you provide tags. But, it can't pick a specific gateway by name.
+
+```yaml
+targetRef:
+  kind: MeshGatewaysSubset
+  tags: {} # not required
+```
+
+Thanks to this we can allow users to target all gateways and apply some good default configuration for them.
```

Comment:
I think we can add a note that this will have to match either only an entire gateway, i.e. ignoring listener tags OR can match specific listeners, depending on the policy.